### DISCUSSION TITLE
What should I set WIREGUARD_ADDRESS to?

### DISCUSSION THREAD

**Discussion by fieu** [qdm12/gluetun#686] [Q&A] [answered]
I was checking the Mullvad WireGuard settings, and sorry if it sounds dumb, but I just don't understand what `WIREGUARD_ADDRESS` means. Is it a random subnet I assign? Is it given to me by Mullvad within their config and I paste it there? I'm confused.

**Comment by qdm12** [comment, marked as answer]
It's your `[Interface]`'s `Address` value (given by Mullvad, and is the same for all servers). So for example `55.66.77.88/32`. And don't worry, it can be confusing if you're not super familiar with Wireguard. I'm sure this discussion will help others 👍

**Reply by fieu** [reply]
Man you are quick! Thank you for clearing it up as well as making gluetun.
